src/api/db/connection.py

```diff
@@ -39,6 +39,8 @@ def execute(self, sql, args, isSELECT=True):
         except psycopg2.Error as e:
             print("DATABASE ERROR: ", end="")
             print(e)
+            self.conn.close()
```

Comment:
i think `self.conn.rollback()` would prolly be better than closing/reconnecting, or we could just run this and open a tech debt issue

Reply:
will give it a testeroo, agree

Reply:
works and logs look happy